### PR TITLE
edit mixed precision user interface

### PR DESCRIPTION
**Pull request by danleifeng**
1. edit mixed precision user interface: offer several params to users.
2. fix bug of warning info.
3. edit fp16 doc: add params description.

### CODE REVIEW

File: plsc/entry.py [hunk 1]
```diff
@@ -300,20 +311,20 @@ def _get_optimizer(self):
...
+
         if self.loss_type in ["dist_softmax", "dist_arcface"]:
             self.optimizer = DistributedClassificationOptimizer(
                 self.optimizer, global_batch_size, use_fp16=self.use_fp16,
                 loss_type=self.loss_type,
                 fp16_user_dict=self.fp16_user_dict)
         elif self.use_fp16:
             self.optimizer = fluid.contrib.mixed_precision.decorate(
-                optimizer=optimizer, init_loss_scaling=self.init_loss_scaling)
+                optimizer=optimizer, init_loss_scaling=self.fp16_user_dict['init_loss_scaling'])
```
Comment (sandyhouse) [inline plsc/entry.py]: How to pass other parameters, e.g., incr_every_n_steps?

File: plsc/models/dist_algo.py [hunk 2]
```diff
@@ -56,12 +55,11 @@ def init_fp16_params(self, loss_type, fp16_user_dict):
...
-                          "in [amp_lists, init_loss_scaling, decr_every_n_nan_or_inf, "
-                          "incr_ratio, decr_ratio, use_dynamic_loss_scaling]." % (key))
+                          "in [init_loss_scaling, incr_every_n_steps, "
+                          "decr_every_n_nan_or_inf, incr_ratio, decr_ratio, "
+                          "use_dynamic_loss_scaling]" % (key))
 
-        self._amp_lists = fp16_params_dict['amp_lists']
-        if self._amp_lists is None:
-            self._amp_lists = AutoMixedPrecisionLists()
+        self._amp_lists = AutoMixedPrecisionLists()
```
Comment (sandyhouse) [inline plsc/models/dist_algo.py]: Why remove this parameter?

File: plsc/entry.py (hunk 1 above)
Comment (danleifeng) [reply plsc/entry.py]: done.thanks!

File: plsc/models/dist_algo.py (hunk 2 above)
Comment (danleifeng) [reply plsc/models/dist_algo.py]: done,thanks!

**Pull request merged**